lib/vet360/contact_information/transaction_response.rb

```diff
@@ -66,8 +66,42 @@ def self.log_error
         log_exception_to_sentry(e)
       end
     end
+
+    class PersonTransactionResponse < TransactionResponse
+      NOT_FOUND_IN_MPI_CODE = 'MVI201'
+
+      def self.from(raw_response, user)
+        return_val = super(raw_response)
+        @user = user
+
+        log_mpi_error if @user.va_profile_status == Common::Client::Concerns::ServiceStatus::RESPONSE_STATUS[:ok]
+
+        return_val
+      end
+
+      def self.log_mpi_error
+        if error?
+          @response_body['tx_messages'].each do |tx_message|
+            if tx_message['code'] == NOT_FOUND_IN_MPI_CODE
+              return log_message_to_sentry(
+                'va profile mpi not found',
+                :error,
+                {
+                  icn: @user.icn,
```

Comment:
I think the icn and edipi both PII that shouldn't be logged to Sentry

Reply:
someone looked it up before and it seems like they are not considered PII
https://github.com/department-of-veterans-affairs/vets-api/pull/2888#issuecomment-474173431

Reply:
Ok, thanks for the link. That was helpful 💯